charts/incubator/oscam/SCALE/questions.yaml

```diff
@@ -0,0 +1,1154 @@
+                              default: 8888
+                              required: true
+                          - variable: nodePort
+                            label: "Node Port (Optional)"
+                            description: "This port gets exposed to the node. Only considered when service type is NodePort"
+                            schema:
+                              type: int
+                              min: 9000
+                              max: 65535
+              - variable: portsList
+                label: "Additional Service Ports"
+                schema:
+                  type: list
+                  default: []
+                  items:
+                    - variable: portsListEntry
+                      label: "Custom ports"
+                      schema:
+                        type: dict
+                        attrs:
+                          - variable: enabled
+                            label: "Enable the port"
+                            schema:
+                              type: boolean
+                              default: true
+                              hidden: true
+                          - variable: name
+                            label: "Port Name"
+                            schema:
+                              type: string
+                              default: ""
+                          - variable: protocol
+                            label: "Port Type"
+                            schema:
+                              type: string
+                              default: "TCP"
+                              enum:
+                                - value: HTTP
+                                  description: "HTTP"
+                                - value: "HTTPS"
+                                  description: "HTTPS"
+                                - value: TCP
+                                  description: "TCP"
+                                - value: "UDP"
+                                  description: "UDP"
+                          - variable: targetPort
+                            label: "Target Port"
+                            description: "This port exposes the container port on the service"
+                            schema:
+                              type: int
+                              required: true
+                          - variable: port
+                            label: "Container Port"
+                            schema:
+                              type: int
+                              required: true
+                          - variable: nodePort
+                            label: "Node Port (Optional)"
+                            description: "This port gets exposed to the node. Only considered when service type is NodePort"
+                            schema:
+                              type: int
+                              min: 9000
+                              max: 65535
```

Comment:
We generally don't include this section:
```suggestion

```

Reply:
The goal of this was, to provide the possibility to expose several ports to the host. This is needed as some ports needs to be available from "outside" (router port forwarding...) to connect to the server. If there is a better way of doing this (except Enable Host Networking) please give me a hint.

Reply:
First off:
If those ports are known, you should add them yourself and not rely on the user adding them.

If it's just random ports they might want, they can already add additionalservices, which offers more options. Additional ports is super niche and not usefull. Adding additional services offers nodeport/loadbalancer selection where addiotnal ports do not.

Reply:
Just for the sake of completeness: those ports are not know, and are freely definable in oscam. Using services works as expected. Thx for the hint!